command/src/main/java/com/jagrosh/jdautilities/command/ContextMenu.java

```diff
@@ -63,6 +63,16 @@ public Map<DiscordLocale, String> getNameLocalization() {
         return nameLocalization;
     }
 
+    /**
+     * {@code true} if the command should always respect user permissions, even if the server overrides them,
+     * {@code false} if the command should ignore user permissions if the server overrides them.
+     * <br>
+     * This defaults to false because it interferes with the server's options for interactions.
+     * <br>
+     * This has no effect for text based commands or DMs.
+     */
+    protected boolean alwaysRespectUserPermissions = false;
```

Comment:
That's quite a mouth full... Maybe a shorter name could be better? Like `forceUserPermissions`, given that, from what I understand, this would force the bot to always check its user perms over server overrides?